pipeline_tools/shared/reference_id.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class ReferenceId(Enum):
```

Comment:
Please add a comment describing where these IDs come from.

Reply:
Could be a link to the ontology lookup service such as https://www.ebi.ac.uk/ols/ontologies/ncbitaxon